docs/benchmarks-scripts/mssql/mssql-script.sql

```diff
@@ -0,0 +1,250 @@
+10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text,
+10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text, 10kb text , 10kb text, 10kb text, 10kb text'''
+
+EXEC insert_rows @allrows = 0, @insertcount = 998, @value = @extraSmallText
+EXEC insert_rows @allrows = 0, @insertcount = 998, @value = @smallText
+EXEC insert_rows @allrows = 0, @insertcount = 998, @value = @regularText
+EXEC insert_rows @allrows = 0, @insertcount = 98, @value = @largeText
+end;
+go --
+
+EXEC table_create @val = 0
+drop procedure if exists insert_rows;
+drop procedure if exists table_create;
+
+EXEC table_copy @tablecount = 1;
```

Comment:
```suggestion
-- TODO: change the value to control the number of tables
EXEC table_copy @tablecount = 1;
```

Instead of taking screenshots, I think it is more update-proof to `TODO`s as comments in the script itself to guide users. In this way, when the script is changed, the screenshots do not need to be updated accordingly.

Reply:
I still cannot put scripts in the specified folders, they will be added only when Eugene's changes are merged